e2e/tsconfig.json

```diff
@@ -3,7 +3,7 @@
   "compilerOptions": {
     "outDir": "../out-tsc/e2e",
     "module": "commonjs",
-    "target": "es5",
+    "target": "es2018",
```

Comment:
Just curious, why the upgrade to `es2018` here?

Reply:
It's automated by `ng update`. I didn't done that manually. I think i can rollback this one change, because fresh angular application have es2015

Reply:
Ok, `es2015` sounds good to me if that works

Reply:
oh i see, this is tsconfig for e2e tests. So it's not relevant imo, because we don't use here e2e tests yet. Application tsconfig is not changed and have `"target": "es2015",`

Reply:
Ah fair enough. No worries then 👍